### PR TITLE
Pending Counter and Recent Transactions list - add pending orders

### PR DESCRIPTION
**Pull request by W3stside**
Adds pending orders to pending counter and activities list

use `addNOrders(10)` in console to test

Overrides several components but as brought up by Anxo + Dima can probably use be extrapolated into 1 custom

### CODE REVIEW

**Comment by W3stside** [comment]
part of #17

File: src/components/AccountDetails/index.tsx [hunk 1]
```diff
@@ -217,7 +217,8 @@ interface AccountDetailsProps {
   ENSName?: string
   openOptions: () => void
 }
-
+const mockPendingTx = ['0x123123123', '0x1235asdasdasd']
```
Comment (Velenir) [inline src/components/AccountDetails/index.tsx:220]: Why changes in `@src/.../AccountDetails` if you override it in `custom/` anyway

**Comment by Velenir** [comment]
Can we have a separate `<PendingOrders>`counter component,instead of overriding the one for Transactions
Lots of copypasta code  with what I assume are minimal changes

Also doyou display counter as `pendingOrders + pendingTxs` or what? Not immediately obvious

**Comment by Velenir** [comment]
Travis, wherefore art thou?
Why you no build?

File: src/components/AccountDetails/index.tsx (hunk 1 above)
Comment (W3stside) [reply src/components/AccountDetails/index.tsx:220]: ah sorry this should be removed!

**Comment by W3stside** [comment]
> Can we have a separate `<PendingOrders>`counter component,instead of overriding the one for Transactions
> Lots of copypasta code with what I assume are minimal changes
> 
> Also doyou display counter as `pendingOrders + pendingTxs` or what? Not immediately obvious

good point, will rework :+1:

**Review by anxolin — COMMENTED** [review]
Great we are tackling this important part. 
Can we please, let's break it a bit in smaller PRs

- If we override a component, let's try to explain why in the description. 
- i.e. `AccountDetails/index.tsx` has a lot of code, ideally it would be nice to know which part is Uni, and which part is not if we really use most of their code.
- There's different ways to override components. One approach we took in other parts (for example see `HeaderMod` and `components/Header/index.tsx` in case that). This way, we can still use some big chunks of the original code
- In general, let's try to make PRs easier to review. It's fine to just do 1 Pr with just the override of a component and no new code (or just tiny hack)

**Comment by W3stside** [comment]
closing in favour of #60

**Pull request closed without merging**